Add Maverick and Biswap v3 on BNB chain to dex.trades

chef-seaweed opened the pull request:
# Thank you for contributing to Spellbook!
Please refer to the top of the `readme` in the root of Spellbook to learn how to contribute to Spellbook on DuneSQL.

Hi team, sorry a bit confused here. For Biswap, since now they have v3 I changed the name of the original `biswap_bnb_trades.sql` to `biswap_v2_bnb_trades.sql` for v2 trades, and added `biswap_v3_bnb_trades.sql` for v3 trades. But as v2 is still SparkSQL(legacy) and v3 is DuneSQL, I'm not sure whether to use `alias = alias('trades')` for `biswap_bnb_trades.sql` and `biswap_trades.sql`. Same happens in Maverick too as `maverick_v1_ethereum_trades.sql` has not been migrated and newly added `maverick_v1_bnb_trades.sql` is in DuneSQL. Thanks

dune-eng commented:
Workflow run id 5527118165 approved.

dune-eng commented:
Workflow run id 5538190130 approved.

dune-eng commented:
Workflow run id 5538483216 approved.

dune-eng commented:
Workflow run id 5538518429 approved.

dune-eng commented:
Workflow run id 5538519405 approved.

jeff-dude commented:
gm @chef-seaweed -- can you confirm your open questions from the description? what can i help with?

chef-seaweed commented:
> > gm @chef-seaweed -- can you confirm your open questions from the description? what can i help with?
> 
> Hi @jeff-dude, I was not sure whether to change aggregated tables of a project (like `maverick.trades` or `biswap.trades`) to DuneSQL when having both DuneSQL and SparkSQL spell upstream cuz when I submitted this PR all existing tables were SparkSQL and only tables I added are DuneSQL.
> 
> I noticed in this PR #3757, Biswap and Maverick seem to have already been migrated to DuneSQL? If this is correct, I can change my commit accordingly, thanks 🙏



> > gm @chef-seaweed -- can you confirm your open questions from the description? what can i help with?
> 
> Hi @jeff-dude, I was not sure whether to change aggregated tables of a project (like `maverick.trades` or `biswap.trades`) to DuneSQL when having both DuneSQL and SparkSQL spell upstream cuz when I submitted this PR all existing tables were SparkSQL and only tables I added are DuneSQL.
> 
> I noticed in this PR #3757, Biswap and Maverick seem to have already been migrated to DuneSQL? If this is correct, I can change my commit accordingly, thanks 🙏

Also kindly ping @couralex6 for confirmation 🙏

jeff-dude commented:
@chef-seaweed if these are new dexs to be added, you should build as dunesql only and wait to integrate into aggregated rollup spells until they are ready. ideally this won't be a long wait, but at least you'll have the new ones built and refreshed on dunesql and can use on the app in the short term until integrated into dex. to confirm, we are migrating the other chain versions of these dexs
